blog/_posts/2021-07-06-distributed-rdataframe.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "RDataFrame is going distributed!"
+layout: archive
+author: Vincenzo Eduardo Padulano
+---
+
+[RDataFrame](https://root.cern/doc/master/classROOT_1_1RDataFrame.html) is ROOT's declarative interface for data
```

Comment:
```suggestion
[RDataFrame](https://root.cern/doc/master/classROOT_1_1RDataFrame.html) is ROOT's high-level interface for data
```

SQL is fully declarative, RDF adopts elements of declarative programming in the API (but you still have to write imperative code for each `Filter` and `Define`). I usually don't get hung up on the details but since we are here :smile: